src/elements/elements.md

```diff
@@ -0,0 +1,124 @@
+# DataJoint Elements for Neurophysiology
+
+<p align="center">
+  <img src="https://raw.githubusercontent.com/datajoint/datajoint.org/0a05cf5c2530a3595a13fc11f6abac64746d845d/static/images/elements-logo.png" width="300">
+</p>  
+
+DataJoint Elements provides an efficient approach for neuroscience labs
+to create and manage _scientific data workflows_: the complex multi-step methods
+for data collection, preparation, processing, analysis, and modeling that
+researchers must perform in the course of an experimental study. The work is
+derived from the developments in leading neuroscience projects and uses the
+[DataJoint API](/core) for defining, deploying, and
+sharing their data workflows.
+
+*DataJoint*
+
+> the open-source framework for data pipelines and automated computational
+> workflows + related documentation, tools, and utilities.
+
+*DataJoint Elements*
+
+> a collection of curated modules for assembling workflows for the major
+> modalities of neurophysiology experiments + related documentation, tools, and
+> utilities.
```

Comment:
Is it necessary to define this here again? Defining it in less places with references has the added benefit of allowing us to be easily modify this in the future and ensure consistency.

Reply:
Good point.  I have removed.